HDFDataset.py

```diff
@@ -276,6 +295,11 @@ def len_info(self):
                       "sequences: %i" % self.num_seqs,
                       "frames: %i" % self.get_num_timesteps()])
 
+  def _get_file_index(self, real_seq_idx):
+    file_index = 0
+    while file_index < len(self.file_start) - 1 and real_seq_idx >= self.file_start[file_index + 1]:
+      file_index += 1
+    return file_index
```

Comment:
Is this function needed? There is `self.file_index` which is used at other places which seems to provide exactly that information.

Reply:
Not anymore, I wanted to save that memory.

Reply:
Ah I see. I guess this is not too much overhead...